tests/transformers/test_transformers_model_export.py

```diff
@@ -1811,6 +1871,52 @@ def test_qa_pipeline_pyfunc_predict(small_qa_pipeline, tmp_path):
     assert values.to_dict(orient="records") == [{0: "Run"}]
 
 
+def test_qa_pipeline_pyfunc_predict_with_kwargs(small_qa_pipeline, tmp_path):
+    artifact_path = "qa_model"
+    inference_payload = json.dumps(
+        {
+            "inputs": {
+                "question": [
+                    "What color is it?",
+                    "How do the people go?",
+                    "What does the 'wolf' howl at?",
+                ],
+                "context": [
+                    "Some people said it was green but I know that it's pink.",
+                    "The people on the bus go up and down. Up and down.",
+                    "The pack of 'wolves' stood on the cliff and a 'lone wolf' howled at "
+                    "the moon for hours.",
+                ],
+            },
+            "top_k": 2,
+            "max_answer_len": 5,
```

Comment:
Can we use a separate field for extra arguments (e.g. `predict_args`)?

Reply:
One limitation of this approach is that we can only support primitive types. Probably not too critical, but something we should consider.